### DISCUSSION TITLE
Cannot preload "name", value of  "modal.column" is undefined

### DISCUSSION THREAD

**Discussion by fatasy** [adonisjs/core#2370] [Help] [answered]
is it possible to ignore the check? or do my own check before calling preload?

![image](https://user-images.githubusercontent.com/18352352/110848504-f1481f00-828c-11eb-9d52-d39790dfc4d2.png)

result: Cannot preload \"chapter\", value of \"library.lastSeenChapterId\" is undefined

do i need to map all the records then check and call the preload method?

**Comment by McSneaky** [comment, marked as answer]
Not possible to ignore it. You need to select all the fields that are required by any joins
